app/models/user.js

```diff
@@ -31,5 +31,13 @@ module.exports = (sequelize, DataTypes) => {
     return `${user.firstName}´s user has been created`;
   };
 
+  User.passwordMatch = function passwordMatch(plainPassword, hashedPassword) {
```

Comment:
use arrow functions, also you are putting the encrypt/decrypt logic inside the models. This shouldnt be here, either put it in the controller or make a new file with bcrypt related functionality.
What should be here are methods related with interacting with the db